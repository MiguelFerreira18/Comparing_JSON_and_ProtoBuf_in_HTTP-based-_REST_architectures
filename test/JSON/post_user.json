{
    "username": "SIMPLEUSER",
    "email": "simpleUser@gmail.com",
    "password":"1234567890",
    "role": "EMPLOYEE"
}