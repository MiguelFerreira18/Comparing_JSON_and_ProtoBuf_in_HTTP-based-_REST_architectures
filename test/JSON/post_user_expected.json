{
    "id": 401,
    "username": "SIMPLEUSER",
    "email": "simpleUser@gmail.com",
    "role": "EMPLOYEE"
}